robocorp-code/src/robocorp_code/inspector/image/_bridge.py

```diff
@@ -0,0 +1,187 @@
+import json
+import os
+import subprocess
+import sys
+import logging
+from pathlib import Path
+from typing import Optional
+from uuid import uuid4
+
+from robocorp_code.inspector.common import (
+    STATE_CLOSED,
+    STATE_INITIALIZING,
+    STATE_PICKING,
+)
+
+DEFAULT_CONFIDENCE = 80
+
+
+def _base64_to_image(img: str):
+    # pylint: disable=import-outside-toplevel
+    from .vendor.utils import base64_to_image
+
+    # making sure we remove the unnecessary composition
+    img = img.replace("data:image/png;base64,", "")
+
+    return base64_to_image(img)
+
+
+class ImageBridge:
+    from robocorp_ls_core.protocols import IEndPoint
```

Comment:
Minor: usually imports should not be inside a class declaration.

Reason: it's mostly the same as a top-level import but the import will now also be available in the class API (which I don't think is what you want in this case).